CONTRIBUTING.md

```diff
@@ -1,9 +1,18 @@
 # Contributing
 
-Lumigator is still in early stages of development and as such will have 
-large variance between even PATCH versions in terms of feature development. 
-While we're in this phase, we're accepting PRs for clarification in documentation. 
+Lumigator is still in early stages of development and as such will have
+large variance between even PATCH versions in terms of feature development.
 
-## Project structure
+We want to encourage both suggestions and contributions from the community in the near future. Before submitting a PR, please open a discussion in a related issue.
```

Comment:
I would remove "in the near future". I don't think we have a crowd looking at our repo so if someone comes and wants to contribute we will find the way to make that happen (if the contribution makes sense)

Reply:
Removed!